### PR TITLE
Adding a hint when there is not internet

### PR DESCRIPTION
**Pull request by harshvitra**
This PR adds a UX hint to tell the user that the internet is not connected. 

Tested in android and ios 

Simulator Screen Shot - iPhone 11 - 2020-02-28 at 00 03 39

### CODE REVIEW

**Comment by harshvitra** [comment]
How to test - 

Close the app. 
Shut down all network connections. 
Start the app, dismiss the red error (Users will not get it)
Visit the pages and you'll see the bar with the message

Now start the wifi / internet
Click on refresh and the bar goes

**Comment by sagararyal** [comment]
Screen Shot 2020-03-02 at 12 02 10 PM
@harshvitra can't seem to make it work on my device.

**Comment by harshvitra** [comment]
> @harshvitra can't seem to make it work on my device.

@sagararyal I tested it worked on my machine, tried to reproduce. 

Can you please do - npm run clean

**Comment by norbertschuler** [comment]
> Can you please do - npm run clean

I once had this issue testing a PR upgrading a lot of npm packages from sanjayradadiya and I also couldn't resolve it any more until I cleaned my development environment completely. So I would agree to the tip @harshvitra suggested.

File: app/containers/Menu/NewBottomNavigator.js [hunk 1]
```diff
@@ -1,10 +1,37 @@
...
 // eslint-disable-next-line react/prefer-stateless-function
 export default class Tabbar extends React.PureComponent {
+  state = {
+    isConnected: true,
+    isInternetReachable: true
+  };
+  checkInternet = () => {
+    const unsubscribe = NetInfo.addEventListener(state => {
+      console.log('Connection type', state.type);
+      console.log('Is connected?', state.isConnected);
```
Comment (norbertschuler) [inline app/containers/Menu/NewBottomNavigator.js]: Please remove `console.log` and replace with `debug`.

**Pull request merged**